config.js

```diff
@@ -14,6 +14,8 @@ module.exports = {
 
   store: {
     app: {
+      language: 'en-US',
+      languages: ['en-US', 'ru', 'uk'],
```

Comment:
`en-US` should be changed to `en`, but in another commit/PR as it will touch a few more files that mention it and aren't related to this feature.

Reply:
I think it makes sense to rename `en-US` to `en` and the added file modifications are OK, I will try to push to this branch but not sure if you gave edit permissions to maintainers on your PR.